core/src/group16/project/game/views/PlaceholderTutorialScreen.kt

```diff
@@ -0,0 +1,64 @@
+package group16.project.game.views
+
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.graphics.Texture
+import com.badlogic.gdx.graphics.g2d.BitmapFont
+import com.badlogic.gdx.graphics.g2d.TextureRegion
+import com.badlogic.gdx.scenes.scene2d.Actor
+import com.badlogic.gdx.scenes.scene2d.ui.Image
+import com.badlogic.gdx.scenes.scene2d.ui.Label
+import com.badlogic.gdx.scenes.scene2d.utils.ChangeListener
+import com.badlogic.gdx.scenes.scene2d.utils.TextureRegionDrawable
+import com.kotcrab.vis.ui.widget.VisLabel
+import com.kotcrab.vis.ui.widget.VisTable
+import com.kotcrab.vis.ui.widget.VisTextButton
+import com.kotcrab.vis.ui.widget.VisTextField
+import group16.project.game.StarBattle
+import group16.project.game.views.components.ImageSlideshowComponent
+import group16.project.game.views.components.PopupComponent
+import group16.project.game.views.components.SlideshowComponent
+
+class PlaceholderTutorialScreen(val gameController: StarBattle) : View() {
```

Comment:
Since these old code snippets were manually added instead of being pulled/merged the versioning might think that these are new files. Should probably remove unused code before merging to master.

Reply:
I thought I merged it

Reply:
Oof, my bad. I guess I overlooked the merge commit. Ignore this message! 😊